l10n/bundle.l10n.json

```diff
@@ -161,8 +161,12 @@
   "Fix All: ": "Fix All: ",
   "C# Workspace Status": "C# Workspace Status",
   "Open solution": "Open solution",
+  "Select context": "Select context",
+  "C# Project Context Status": "C# Project Context Status",
+  "Active File Context": "Active File Context",
```

Comment:
Wondering - could we merge this into the select button?  
e.g.
```ProjectA (net6.0)            Change Active File Context```

or maybe we could even put the document name in it?
```ProjectA (net6.0)            Change Program.cs Context```

Reply:
Sure, although the select command is only visible when there are multiple project contexts. Would it be jarring for the text to move from the item details to the command and back again as you navigate?

Reply:
ah hmm.  I wonder if we should just always show the button even if there's only one.  Not sure